src/components/chips/chips.scss

```diff
@@ -0,0 +1,39 @@
+$chip-bgcolor: #F5F5F5;
+$chip-border: 1px solid #DCDCDC;
+$chip-font-size: 13px;
+$chip-height: 22px;
+$chip-button-padding: 2px;
+$chip-padding: 1px 5px 0px 5px;
+$chip-margin: 0 0 0 4px;
+
+.md-chips {
+  * {
+    font-family: $font-family;
+    font-size: $chip-font-size;
+  }
+  li:first-child {
```

Comment:
Try to avoid using `ul` and `li` and just use CSS classes in rules. 

Also, try to reduce the amount of nesting going on here.